lib/D2L.Extensibility.AuthSdk/Impl/D2LUserContext.cs

```diff
@@ -17,10 +17,18 @@ internal sealed class D2LUserContext : ID2LUserContext {
         }
 
         private Uri CreateApiUrl(string path) {
-
+			

```

Comment:
Thanks for catching it. Visual Studio is driving me :rage:. I set the whitespaces to be consistent in commit 4eddb53.